### DISCUSSION TITLE
Still supported?

### DISCUSSION THREAD

**Discussion by Dvalin21** [louislam/dockge#559] [General]
Is dockge still supported?

**Comment by hobybrenner** [comment]
I sure hope so, this software is has been awesome from the beginning, and now with Portainer tightening the reigns on its home license use, I suspect a lot more homelabber will be looking for something like DockGe

**Reply by Dvalin21** [reply]
I absolutely agree!! Thats why its a concern. I've ditched portainer some months ago and from the sound of it, it was a good choice. Matter of fact I don't even hear most of the youtubers talk about it as much anymore.

**Reply by cyril59310** [reply]
for now, Louislam is focusing on version 2.0 of his other project, [Uptime Kuma](https://github.com/louislam/uptime-kuma). He will return to this project afterward

**Reply by Dvalin21** [reply]
> for now, Louislam is focusing on version 2.0 of his other project, [Uptime Kuma](https://github.com/louislam/uptime-kuma). He will return to this project afterward

Thats all I wanted to know, thanks.
